[2022-12-04 16:02:19,825] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.hel1o scheduled__2022-02-11T00:00:00+00:00 [queued]>
[2022-12-04 16:02:20,845] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.hel1o scheduled__2022-02-11T00:00:00+00:00 [queued]>
[2022-12-04 16:02:20,882] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:02:20,898] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-04 16:02:20,966] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 16:02:23,054] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): hel1o> on 2022-02-11 00:00:00+00:00
[2022-12-04 16:02:23,170] {standard_task_runner.py:52} INFO - Started process 422 to run task
[2022-12-04 16:02:23,692] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'hel1o', 'scheduled__2022-02-11T00:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpxclevbn0', '--error-file', '/tmp/tmpcxrb_bxl']
[2022-12-04 16:02:23,734] {standard_task_runner.py:80} INFO - Job 50: Subtask hel1o
[2022-12-04 16:02:25,359] {task_command.py:369} INFO - Running <TaskInstance: first_dag.hel1o scheduled__2022-02-11T00:00:00+00:00 [running]> on host fb90b48d1471
[2022-12-04 16:02:26,675] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=hel1o
AIRFLOW_CTX_EXECUTION_DATE=2022-02-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-11T00:00:00+00:00
[2022-12-04 16:02:26,690] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-12-04 16:02:26,713] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hello']
[2022-12-04 16:02:27,019] {subprocess.py:85} INFO - Output:
[2022-12-04 16:02:27,067] {subprocess.py:92} INFO - hello
[2022-12-04 16:02:27,075] {subprocess.py:96} INFO - Command exited with return code 0
[2022-12-04 16:02:29,386] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=hel1o, execution_date=20220211T000000, start_date=20221204T160220, end_date=20221204T160229
[2022-12-04 16:02:29,813] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-04 16:02:30,819] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
